packages/build-utils/test/fixtures/20-npm-7/vercel.json

```diff
@@ -1,5 +1,5 @@
 {
   "version": 2,
   "builds": [{ "src": "package.json", "use": "@vercel/static-build" }],
-  "probes": [{ "path": "/", "mustContain": "npm version: 7" }]
+  "probes": [{ "path": "/", "mustContain": "npm version: 8" }]
```

Comment:
@TooTallNate Looks like [Node.js 16](https://github.com/nodejs/node/releases/tag/v16.12.0) actually bumped a npm major. I don't think this will cause any problems but wanted to point it out.